docs/source/uploader.rst

```diff
@@ -123,6 +123,46 @@ You can override the most validation configuration values set in ``config/mediab
 
         ->upload();
 
+You can also validate the file without uploading it by calling the ``verifyFile`` method, it will always return true or throw a ``Plank\Mediable\MediaUploadException``
```

Comment:
I know I was the one that recommended true or exception, but on reflection that is a bit silly. Should be void or Exception.

You can update this line to say 
> If the file does not pass validation, an instance of Plank\Mediable\MediaUploadException will be thrown